cmd/sealer/cmd/cluster/cmd.go

```diff
@@ -0,0 +1,100 @@
+// Copyright © 2021 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cluster
+
+import (
+	"github.com/sealerio/sealer/cmd/sealer/cmd/types"
+	"github.com/spf13/cobra"
+)
+
+// cmd flags
+var (
+	runFlags      = &types.RunFlags{}
+	applyFlags    = &types.ApplyFlags{}
+	scaleUpFlags  = &types.ScaleUpFlags{}
+	joinFlags     = &types.ScaleUpFlags{}
+	rollbackFlags = &types.RollbackFlags{}
+	upgradeFlags  = &types.UpgradeFlags{}
+	deleteFlags   = &types.DeleteFlags{}
+)
+
+// long cmd cli message
+var (
```

Comment:
In fact, I personally think it might be better for each cmd to maintain all of its own information so that fewer modifications are involved when we add or deprecated a command

Reply:
yes,  we do better keep each description with its own cmd. but if we have much more subcommand , aggregated them together may more easier to maintain.

Reply:
I don't think so, unless you have a need to batch modify subcommands